ovs/lib/tests/vdisk_tests/test_get_set_config_params.py

```diff
@@ -49,7 +49,7 @@ def setUpClass(cls):
         fakesleep.monkey_patch()
         EtcdConfiguration.set('/ovs/framework/storagedriver|mds_tlogs', 100)
         EtcdConfiguration.set('/ovs/framework/storagedriver|mds_maxload', 75)
-        EtcdConfiguration.set('/ovs/framework/storagedriver|mds_safety', 2)
+        EtcdConfiguration.set('/ovs/framework/storagedriver|mds_safety', 3)
```

Comment:
It's a unittest, there are no volumes or mds services actually running, the test just validates the behavior of the framework's decisions. It might indeed be a good idea to revert these two test changes, as there are separate tests that validate the mds_safety behavior.